docs/scaler/modules/appservice.md

```diff
@@ -10,6 +10,7 @@ This README will describe how to use Bellhop to help scale your Azure App servic
 setState-WorkerSize = <String>
 setState-Tier = <String>
 ```
+NOTE: User must provide either the `WorkerSize` or `Tier` tag, or both. 
```

Comment:
fantastic! I didn't know this worked.

Reply:
Neither did @CloudViking and I until we tested it, so it is now noted in the docs!